dockerfiles/jupyter/Dockerfile

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 #
 
-ARG MLRUN_PYTHON_VERSION=3.9
+ARG MLRUN_PYTHON_VERSION=3.11
 ARG MLRUN_UV_IMAGE=ghcr.io/astral-sh/uv:latest
 
 FROM ${MLRUN_UV_IMAGE} AS uv-image
 
-FROM quay.io/mlrun/jupyter-scipy-notebook:python-${MLRUN_PYTHON_VERSION}
+FROM quay.io/jupyter/scipy-notebook:python-${MLRUN_PYTHON_VERSION}
```

Comment:
we should push the base image to `quay.io/mlrun` to make sure it doesn't change under our feet and to avoid rate limiting